ycmd/completers/go/tests/gocode_completer_test.py

```diff
@@ -84,10 +84,14 @@ def ComputeCandidatesInner_test( self ):
     result = self._completer.ComputeCandidatesInner(self._BuildRequest(10, 40))
     eq_(mock.cmd, ['THE_BINARY', '-f=json', 'autocomplete', PATH_TO_TEST_FILE, "292"])
     eq_(result, [{
-        'menu_text': u'Prefix',
-        'insertion_text': u'Prefix',
-        'extra_menu_info': u'func() string',
-        'kind': u'func'
+      'typed_string': u'Prefix',
```

Comment:
Changing all these names changes the public API; we'll have to update YCM itself plus notify emacs-ycmd people like @abingham if we go this route. All of that work may not be worth it.